tests/test_grains_minicircle.py

```diff
@@ -45,6 +45,24 @@ def test_gaussian_filter_minicircle(minicircle_grain_gaussian_filter: Grains, pl
     return fig
 
 
+@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
+def test_z_threshold_minicircle(minicircle_grain_z_threshold: Grains, plotting_config: dict, grain_config: dict, tmpdir) -> None:
```

Comment:
Minor but you can remove `grain_config: dict` here as its no longer used.

Reply:
Cool, I've removed `grain_config` and used `minicircle_grain_z_threshold.zrange[X]` instead